gpjax/gps.py

```diff
@@ -44,23 +82,27 @@ class Prior(GP):
     kernel: Kernel
     mean_function: tp.Optional[MeanFunction] = Zero()
     name: tp.Optional[str] = "Prior"
+    jitter: tp.Optional[float] = DEFAULT_JITTER 
```

Comment:
I'm not sure that passing the kernel's jittter amount into the GP instantiator is particularly helpful. It may be best left as a global parameter.